src/RA_Dlg_AchievementsReporter.cpp

```diff
@@ -30,78 +27,86 @@ void Dlg_AchievementsReporter::SetupColumns(HWND hList)
     //	Remove all data.
     ListView_DeleteAllItems(hList);
 
-    LV_COLUMN col;
-    ZeroMemory(&col, sizeof(col));
-
-    for (size_t i = 0; i < SIZEOF_ARRAY(COL_TITLE); ++i)
+    auto i{ 0 };
+    for (const auto& sTitle : COL_TITLE)
     {
-        col.mask = LVCF_TEXT | LVCF_WIDTH | LVCF_SUBITEM | LVCF_FMT;
-        col.cx = COL_SIZE[i];
-        col.cchTextMax = 255;
-        ra::tstring str = NativeStr(COL_TITLE[i]);	//	Hold the temporary object
-        col.pszText = str.data();
-        col.iSubItem = i;
-
-        col.fmt = LVCFMT_LEFT | LVCFMT_FIXED_WIDTH;
-        if (i == SIZEOF_ARRAY(COL_TITLE) - 1)	//If the last element: fill to the end
+        ra::tstring str{ sTitle };
+        LV_COLUMN col
+        {
+            col.mask       = ra::to_unsigned(LVCF_TEXT | LVCF_WIDTH | LVCF_SUBITEM | LVCF_FMT),
+            col.fmt        = LVCFMT_LEFT | LVCFMT_FIXED_WIDTH,
+            col.cx         = COL_SIZE.at(i),
+            col.pszText    = str.data(),
+            col.cchTextMax = 255,
+            col.iSubItem   = i
+        };
+
+        if (i == ra::to_signed(COL_TITLE.size() - 1))	//If the last element: fill to the end
```

Comment:
The intent of `i` is to be unsigned. Recommend changing it back to a size_t, and using `to_unsigned` on line 41.